src/mono/mono/metadata/class-setup-vtable.c

```diff
@@ -1065,7 +1065,7 @@ handle_dim_conflicts (MonoMethod **vtable, MonoClass *klass, GHashTable *conflic
 		int nentries = 0;
 		MonoMethod *impl = NULL;
 		for (l = entries; l; l = l->next) {
-			if (l->data) {
+			if (l->data && l->data != impl) {
 				nentries ++;
```

Comment:
How is this related to the PR ?

Reply:
I stumbled across this bug after changing the invoke machinery. I don't remember exactly why the test was passing before.